src/components/Tooltip/index.tsx

```diff
@@ -3,10 +3,11 @@ import styled from 'styled-components'
 import { Button } from 'ariakit/button'
 import { Tooltip as AriaTooltip, TooltipAnchor, useTooltipState } from 'ariakit/tooltip'
 
-interface ITooltip {
+interface ITooltip<RenderAs = any> {
 	content: string | null
 	style?: {}
 	children: React.ReactNode
+	as?: RenderAs
```

Comment:
default `as` is `TooltipTrigger` = `Button`
we need `as` extensibility, because we don't want to always have button triggers

e.g. it breaks layout in the `RecentProtocols` table

(I hope it's fine that I use non-single-letter generics, but I think it's just cleaner)